sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/api/Azure.Iot.TimeSeriesInsights.netstandard2.0.cs

```diff
@@ -1,32 +1,32 @@
-namespace Azure.Iot.TimeSeriesInsights
+namespace Azure.IoT.TimeSeriesInsights
```

Comment:
the name of this file still shows up with lowercase t here.
![image](https://user-images.githubusercontent.com/18057016/112188683-6df9c800-8bc0-11eb-8b3a-0184a67ed9d8.png)

Reply:
It's showing up differently on github but if you hover over the name of file it shows correct casing